app/controllers/photos_controller.rb

```diff
@@ -65,18 +65,23 @@ def make_profile_photo
     @photo = Photo.where(:id => params[:photo_id], :author_id => author_id).first
 
     if @photo
-      profile_hash = {:image_url        => @photo.url(:thumb_large),
-                      :image_url_medium => @photo.url(:thumb_medium),
-                      :image_url_small  => @photo.url(:thumb_small)}
+      profile_hash = {image_url: @photo.url(:thumb_large),
+                      image_url_medium: @photo.url(:thumb_medium),
+                      image_url_huge: @photo.url(:thumb_huge),
+                      image_url_small: @photo.url(:thumb_small)}
 
       if current_user.update_profile(profile_hash)
         respond_to do |format|
-          format.js{ render :json => { :photo_id  => @photo.id,
-                                       :image_url => @photo.url(:thumb_large),
-                                       :image_url_medium => @photo.url(:thumb_medium),
-                                       :image_url_small  => @photo.url(:thumb_small),
-                                       :author_id => author_id},
-                            :status => 201}
+          format.js{ render json: {
+                                    photo_id: @photo.id,
+                                    image_url: @photo.url(:thumb_large),
+                                    image_url_huge: @photo.url(:thumb_huge),
+                                    image_url_medium: @photo.url(:thumb_medium),
+                                    image_url_small: @photo.url(:thumb_small),
+                                    author_id: author_id
+                                  },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.